_includes/doc-registry.html

```diff
@@ -0,0 +1,31 @@
+{%- assign documentation_navigation=site.documentation_navigation -%}
+{%- for nav_item in documentation_navigation -%}
+  <h2> {%- if nav_item.icon -%}
+  <i class='fa fa-fw {{ nav_item.icon }}'></i>
```

Comment:
these are the font awesome icon classes. Was it not possible to migrate these to USWDS icons?

Reply:
I don't even see a reference to this template. Deleting the file.